docs/source/format/Metadata.rst

```diff
@@ -192,13 +189,74 @@ categories:
 
 Refer to `Schema.fbs`_ for up-to-date descriptions of each built-in
 logical type.
+Custom Application Metadata
```

Comment:
An empty new line is missing before this line.

Reply:
fixed